lib/internal/async_hooks.js

```diff
@@ -127,6 +127,15 @@ function fatalError(e) {
   process.exit(1);
 }
 
+function lookupPublicResource(resource) {
+  if (typeof resource !== 'object') return resource;
```

Comment:
Nit: I don't think it is possible but pointing out that this doesn't check against `resource === null`.

Reply:
@lundibundi Yeah, thanks for pointing that out… I agree it shouldn’t happen but I’ve still added the check here for correctness